### ISSUE TITLE
could not find counter "Clock Interrupts/sec"

### ISSUE THREAD

**Issue by irvintim** [martinlindhe/wmi_exporter#344]
Running the 0.7,999 preview release on Windows Server 2008 R2 Standard, I am unable to get CPU stats, the error I receive is shown below.

```
PS C:\wmi_exporter> .\wmi_exporter-0.7.999-preview.2-amd64.exe --collectors.enabled="cpu,memory,net,logical_disk,os,syst
em"
←[36mINFO←[0m[0003] Enabled collectors: memory, net, logical_disk, os, system, cpu  ←[36msource←[0m="exporter.go:313"
←[36mINFO←[0m[0003] Starting WMI exporter (version=0.7.999-preview.2, branch=timeout-scrapes, revision=daa6f3d111ba34a9e
b64cdfbe5160192b4566e1a)  ←[36msource←[0m="exporter.go:327"
←[36mINFO←[0m[0003] Build context (go=go1.12.3, user=appvyr-win\appveyor@APPVYR-WIN, date=20190525-12:49:02)  ←[36msourc
e←[0m="exporter.go:328"
←[36mINFO←[0m[0003] Starting server on :9182                      ←[36msource←[0m="exporter.go:331"
←[31mERRO←[0m[0006] collector cpu failed after 0.000977s: could not find counter "Clock Interrupts/sec" on instance  ←[3
1msource←[0m="exporter.go:172"
←[31mERRO←[0m[0014] collector cpu failed after 0.000000s: could not find counter "Clock Interrupts/sec" on instance  ←[3
1msource←[0m="exporter.go:172"
←[31mERRO←[0m[0022] collector cpu failed after 0.000000s: could not find counter "Clock Interrupts/sec" on instance  ←[3
1msource←[0m="exporter.go:172"
```
Here is the relevant section from the output from the wmi_exporter URL:
```
# HELP wmi_exporter_collector_duration_seconds wmi_exporter: Duration of a collection.
# TYPE wmi_exporter_collector_duration_seconds gauge
wmi_exporter_collector_duration_seconds{collector="cpu"} 0
wmi_exporter_collector_duration_seconds{collector="logical_disk"} 0.4492188
wmi_exporter_collector_duration_seconds{collector="memory"} 0.0673828
wmi_exporter_collector_duration_seconds{collector="net"} 0.3769531
wmi_exporter_collector_duration_seconds{collector="os"} 0.515625
wmi_exporter_collector_duration_seconds{collector="system"} 0.5566406
# HELP wmi_exporter_collector_success wmi_exporter: Whether the collector was successful.
# TYPE wmi_exporter_collector_success gauge
wmi_exporter_collector_success{collector="cpu"} 0
wmi_exporter_collector_success{collector="logical_disk"} 1
wmi_exporter_collector_success{collector="memory"} 1
wmi_exporter_collector_success{collector="net"} 1
wmi_exporter_collector_success{collector="os"} 1
wmi_exporter_collector_success{collector="system"} 1
# HELP wmi_exporter_collector_timeout wmi_exporter: Whether the collector timed out.
# TYPE wmi_exporter_collector_timeout gauge
wmi_exporter_collector_timeout{collector="cpu"} 0
wmi_exporter_collector_timeout{collector="logical_disk"} 0
wmi_exporter_collector_timeout{collector="memory"} 0
wmi_exporter_collector_timeout{collector="net"} 0
wmi_exporter_collector_timeout{collector="os"} 0
```

**Comment by carlpett** [comment]
Hi @irvintim,
Thanks for the report, and thanks for trying it out! Could you run this Powershell command and paste the output here?
```powershell
Get-Counter -ListSet "Processor Information" | Select -ExpandProperty Counter
```

**Comment by irvintim** [comment]
Here you go -- thanks for looking into it!

```
PS C:\Users\root> Get-Counter -ListSet "Processor Information" | Select -ExpandProperty Counter
\Processor Information(*)\Processor State Flags
\Processor Information(*)\% of Maximum Frequency
\Processor Information(*)\Processor Frequency
\Processor Information(*)\Parking Status
\Processor Information(*)\% Priority Time
\Processor Information(*)\C3 Transitions/sec
\Processor Information(*)\C2 Transitions/sec
\Processor Information(*)\C1 Transitions/sec
\Processor Information(*)\% C3 Time
\Processor Information(*)\% C2 Time
\Processor Information(*)\% C1 Time
\Processor Information(*)\% Idle Time
\Processor Information(*)\DPC Rate
\Processor Information(*)\DPCs Queued/sec
\Processor Information(*)\% Interrupt Time
\Processor Information(*)\% DPC Time
\Processor Information(*)\Interrupts/sec
\Processor Information(*)\% Privileged Time
\Processor Information(*)\% User Time
\Processor Information(*)\% Processor Time
```

**Comment by carlpett** [comment]
Thanks! This explains it, apparently not all the counters in the docs were made available at once.
We probably shouldn't fail the collector in these cases, but just let the metric be zero. I'll make a fix for that.

**Comment by carlpett** [comment]
This is fixed now, will be in the 0.8 release.
